<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="true">
 <class>FactoryArea</class>
 <widget class="QFrame" name="FactoryArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="FactoryLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Equipment Development</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Slots">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="SlotGrid">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="FactorySlot" name="Factory_Slot_0">
          <property name="text">
           <string>Factory 0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="FactorySlot" name="Factory_Slot_6">
          <property name="text">
           <string>Factory 6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="FactorySlot" name="Factory_Slot_12">
          <property name="text">
           <string>Factory 12</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="FactorySlot" name="Factory_Slot_18">
          <property name="text">
           <string>Factory 18</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="FactorySlot" name="Factory_Slot_1">
          <property name="text">
           <string>Factory 1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="FactorySlot" name="Factory_Slot_7">
          <property name="text">
           <string>Factory 7</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="FactorySlot" name="Factory_Slot_13">
          <property name="text">
           <string>Factory 13</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="FactorySlot" name="Factory_Slot_19">
          <property name="text">
           <string>Factory 19</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="FactorySlot" name="Factory_Slot_2">
          <property name="text">
           <string>Factory 2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="FactorySlot" name="Factory_Slot_8">
          <property name="text">
           <string>Factory 8</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="FactorySlot" name="Factory_Slot_14">
          <property name="text">
           <string>Factory 14</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="FactorySlot" name="Factory_Slot_20">
          <property name="text">
           <string>Factory 20</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="FactorySlot" name="Factory_Slot_3">
          <property name="text">
           <string>Factory 3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="FactorySlot" name="Factory_Slot_9">
          <property name="text">
           <string>Factory 9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="FactorySlot" name="Factory_Slot_15">
          <property name="text">
           <string>Factory 15</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="FactorySlot" name="Factory_Slot_21">
          <property name="text">
           <string>Factory 21</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="FactorySlot" name="Factory_Slot_4">
          <property name="text">
           <string>Factory 4</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="FactorySlot" name="Factory_Slot_10">
          <property name="text">
           <string>Factory 10</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="FactorySlot" name="Factory_Slot_16">
          <property name="text">
           <string>Factory 16</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="FactorySlot" name="Factory_Slot_22">
          <property name="text">
           <string>Factory 22</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="FactorySlot" name="Factory_Slot_5">
          <property name="text">
           <string>Factory 5</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="FactorySlot" name="Factory_Slot_11">
          <property name="text">
           <string>Factory 11</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="FactorySlot" name="Factory_Slot_17">
          <property name="text">
           <string>Factory 17</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="FactorySlot" name="Factory_Slot_23">
          <property name="text">
           <string>Factory 23</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ArsenalArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>255</horstretch>
       <verstretch>255</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FactorySlot</class>
   <extends>QPushButton</extends>
   <header>FactorySlot/factoryslot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
